src/pretix/static/pretixcontrol/js/ui/main.js

```diff
@@ -614,7 +614,10 @@ $(function () {
         $('.nav-tabs a[href="#' + url.split('#')[1] + '"]').tab('show');
     }
     $('a[data-toggle="tab"]').on('click', function (e) {
-        window.location.hash = e.target.hash;
+        if (!$(this).closest(".panel").length) {
```

Comment:
Do we really want to check for ``.panel`` or rather for ``.tab-content``? The location.hash thing is bound to cause problems whenever tabs are nested, regardless of whether there are panels involved.

Reply:
Yes, that makes total sense!